Gemfile

```diff
@@ -12,7 +12,7 @@ gem 'logstasher', '1.2.2'
 gem 'mongoid', '~> 6.0.0'
 gem 'pundit'
 gem 'sass-rails', '~> 5.0.4'
-gem 'select2-rails', '~> 4.0.3'
+gem 'select2-rails', '~> 3.5.10'
```

Comment:
We should probably comment why we keeping this version

Reply:
Good point. I've added a comment.

Reply:
Is there a magic comment we can add to the gemfile for telling dependabot not to bother us for this gem?

Reply:
When you close a dependabot PR, it gives you instructions on how to tell it to ignore a minor or major version. I didn't do with this gem in case someone wanted to try the upgrade process, which might be overly optimistic. 🙂